README.md

```diff
@@ -0,0 +1,226 @@
+</td>
+<td><code>oidc-server</code></td>
+</tr>
+<tr>
+  <td>Issuer</td>
+  <td>Uri</td>
+  <td>The base address (absolute) used to uniquely identify the authorization server.<br>
+      <sub><i>Unless <code>AllowInsecureHttp</code> has been set to <code>true</code>, an HTTPS address must be provided.</i></sub>
+  </td>
+  <td>Automatically inferred from the request URL</td>
+</tr>
+<tr>
+  <td>
+    AuthorizationEndpointPath
+    <br>
+    LogoutEndpointPath
+  </td>
+  <td>PathString</td>
+  <td>The path of the endpoint. Can be set to <code>PathString.Empty</code> to disable the endpoint.</td>
+  <td>
+      <code>/connect/authorize</code>
+      <br>
+      <code>/connect/logout</code>
+  </td>
+      
+</tr>
+<tr>
+  <td>
+    AuthorizationCodeLifetime
+    <br>
+    AccessTokenLifetime
+    <br>
+    IdentityTokenLifetime
+    <br>
+    RefreshTokenLifetime
+  </td>
+  <td>TimeSpan</td>
+  <td>The period of time the token or code remains valid after being issued.</td>
+  <td>
+    5 minutes
+    <br>
+    1 hour
+    <br>
+    20 minutes
+    <br>
+    6 hours
+  </td>
+</tr>
+<tr>
+  <td>UseSlidingExpiration</td>
+  <td>bool</td>
+  <td>
+    Determines whether refresh tokens issued during a <code>grant_type=refresh_token</code> request should be generated with a new expiration date or should re-use the same expiration date as the original refresh token.
+    <br>
+    <sub>Set this property to <code>true</code> to assign a new expiration date each time a refresh token is issued, <code>false</code> to use the expiration date of the original refresh token.</sub>
+  </td>
+  <td><code>true</code></td>
+</tr>
+<tr>
+  <td>ApplicationCanDisplayErrors</td>
+  <td>bool</td>
+  <td>Set to <code>true</code> if the web application is able to render error messages on the authorization endpoint.</td>
+  <td><code>false</code></td>
+</tr>
+<tr>
+  <td>AllowInsecureHttp</td>
+  <td>bool</td>
+  <td>
+    Set to <code>true</code> to allow incoming requests to arrive on HTTP and to allow `redirect_uri` parameters to have HTTP URI addresses.
+    <br>
+    <sub>Setting this option to <code>false</code> in production is strongly encouraged to mitigate man-in-the-middle attacks.</sub>
+  </td>
+  <td><code>false</code></td>
+</tr>
+<tr>
+  <td>UseCustomViews</td>
+  <td>bool</td>
+  <td>
+  OpenIddict comes with built in views out of the box. If you want to use your custom views, you can set this option to true and provide the following required views:
+  <br>
+  <ul>
+    <li>Authorize.cshtml</li>
+    <li>Logout.cshtml</li>
+    <li>Signin.cshtml</li>
+    <li>Error.cshtml</li>
+  </ul>
+  </td>
+  <td><code>false</code></td>
+</tr>
+</tbody>
+</table>
+
+You can find working samples in the [samples](https://github.com/openiddict/core/tree/dev/samples) directory.
+
+Nightly builds can now be found on the [aspnet-contrib](https://github.com/aspnet-contrib) MyGet repository https://www.myget.org/F/aspnet-contrib/api/v3/index.json.
```

Comment:
Yeah, move this one at the very begging of the `Getting started` section :smile: